tests/unit/suites/plugins/filesystem/local/adapter/MediaFileAdapterLocalTest.php

```diff
@@ -359,4 +359,131 @@ public function testDeleteInvalidPath()
 		// Fetch the file from the root folder
 		$adapter->delete('invalid');
 	}
+
+	public function testCopy()
```

Comment:
Will separate methods instead and check them, i wrote methods to handle anykind of copying folders and files, but it seems better to separate them for better readability